romancal/jump/jump_step.py

```diff
@@ -85,21 +86,22 @@ def process(self, input):
             # Get the gain and readnoise reference files
             gain_filename = self.get_reference_file(input_model, 'gain')
             self.log.info('Using GAIN reference file: %s', gain_filename)
-            gain_model = rdm.GainModel(gain_filename)
+            gain_model = rdd.GainRefModel(gain_filename)
```

Comment:
Note: this will not actually convert the file to a GainModel. Accordingly, this can't be relied upon to make non-gain input to work (if that is the desired effect).